models/document.py

```diff
@@ -83,6 +92,21 @@ def load_collection(self):
                 )
 
 
+class StanfordDocumentCollection(DocumentCollection):
+    def load_collection(self):
+        for directory in os.listdir(self.data_filename):
+            print("DIRECTORY", directory)
```

Comment:
You keep this print, is it normal?

Reply:
As the parsing of the collection can be quite long (up to 10 mins), I find it useful and not too verbose to be sure the parsing goes on. Do you think we should remove it ?

Reply:
No I think it a good solution, I just wanted to be sure we know why we keep a print in the model :)